test-cases/gutenberg/columns.md

```diff
@@ -1,19 +0,0 @@
-# Columns Block - Test Cases
-
---------------------------------------------------------------------------------
-
-##### TC010
-
-### Columns percentage mechanism - custom percentage values are rendered correctly on frontend
```

Comment:
Other automated Columns tests assert the expected mark up. I think it is safe to presume that if the mark up is correct, the front-end website will render correctly in the browser.

Reply:
Yep, as pointed out, we are already checking the HTML content related to the column's width ([reference](https://github.com/WordPress/gutenberg/blob/58f5e9a6a972d16aeca13693ce863bcedaf342b0/packages/block-library/src/columns/test/edit.native.js#L338-L431)). So this test case is not really adding value to the test coverage 👍 .